src/Goerli-DssSpell.sol

```diff
@@ -42,64 +42,255 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
 
-    // --- Rates ---
-    uint256 constant ZERO_FIVE_PCT_RATE      = 1000000000158153903837946257;
-    uint256 constant TWO_FIVE_PCT_RATE       = 1000000000782997609082909351;
-    uint256 constant THREE_PCT_RATE          = 1000000000937303470807876289;
+    address constant SB_LERP = address(0x98bbDDe76EB5753578280bEA86Ed8401f2831213);
+    address constant NEW_MCD_ESM = address(0x023A960cb9BE7eDE35B433256f4AfE9013334b55);
+    bytes32 constant MCD_ESM = "MCD_ESM";
```

Comment:
Don't think this adds any value, would prefer to just use "MCD_ESM" inline.

Reply:
So strictly speaking we should be limiting the number of times we print out tags to reduce the likelihood of homograph errors/attacks.

For example:

```
bytes32("MCD_ESM")
0x4d43445f45534d000000000000000000000000000000000000000000000000

bytes32("МCD_ESM")
0xd09c43445f45534d0000000000000000000000000000000000000000000000
```

If we use a tag once we can inline it, but for multiple uses of a tag we probably just want to use it once so we ensure that all instances use the same bytes32 string.

Reply:
Sure not a big deal either way.